crates/defguard_core/src/db/models/mod.rs

```diff
@@ -212,11 +215,16 @@ impl UserDetails {
     pub async fn from_user(pool: &PgPool, user: &User<Id>) -> Result<Self, SqlxError> {
         let devices = user.user_devices(pool).await?;
         let security_keys = user.security_keys(pool).await?;
-
+        let biometric_enabled_devices: Vec<i64> = BiometricAuth::find_by_user_id(pool, user.id)
```

Comment:
```suggestion
        let biometric_enabled_devices = BiometricAuth::find_by_user_id(pool, user.id)
```
eventually add `collect::<Vec<_>>()`